### DISCUSSION TITLE
Loading Undetected Chromedriver with an extension

### DISCUSSION THREAD

**Discussion by tablescrapers** [ultrafunkamsterdam/undetected-chromedriver#1160] [Q&A]
How to I load an extension using options cos I tried the way u would do it in normal selenium and it works fine but when I try in undetected chromedriver it don't load in.

**Comment by nordinho** [comment]
You have to upload it like this, not with crx file:

options.add_argument("--load-extension=" + r"C:\Users\Username\AppData\Local\Google\Chrome\User Data\Default\Extensions\fjoaledfpmneenckfbpdfhkmimnjocfa\2.73.1_0")

Let me know if you manage to upload more than 1 extension.
